PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -101,10 +101,27 @@ public final class DefaultControlsViewController: ContentControlsViewController
         }
     }
     
-    var isApperared = false
+    private var controllerStates = ControllerStates()
```

Comment:
`States`? Is it collection?

Reply:
@BogdanBilonog how should I call it then?

Reply:
@RomanTysiachnik  `ControllerState`?

Reply:
@BogdanBilonog fixed